File: Programming Language(ver.2)/PHP/405.php
<?php
    $array_index = new SplFixedArray(3);
    $array_sentence = new SplFixedArray(3);
    $file = fopen("read.txt", "r") or die("Unable to open file");
    for($i = 0; $i < 3; $i++){
        $array_index[$i] = intval(fgets(STDIN));
    }
    for($i = 0; $i < 3; $i++){
        $array_sentence[$i] = trim(fgets($file));
    }
    for($i = 0; $i < 3; $i++){
        $result = "";
        for($j = 0; $j < $array_index[$i]; $j++){
            $result .= strval($array_index[$i]);
        }
        for($j = $array_index[$i]; $j < strlen($array_sentence[$i]); $j++){
            $result .= $array_sentence[$i][$j]; 
        }
        printf("%s\n", $result);
    }
?>

File: Programming Language(ver.2)/C/405.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main ()
{
    FILE * input;
    char str[1024];
    int a[3];
    int i;
    for(i = 0;i < 3;i++)
        scanf("%d",&a[i]);

    input = fopen("read.txt","rt");

    for(i = 0;!feof(input);i++){
        if(fgets(str,sizeof(str),input) != NULL){
            memset(str,a[i] + '0',a[i]);
            printf(str);
        }
    }
    fclose(input);
    return 0;
}
